doc/pod-creation-example.md

```diff
@@ -0,0 +1,62 @@
+# Example - Allocating MAC Addresses to a Pod
+
+## Create a network-attachment-definition:
```

Comment:
Colon is redundant, title has it implicitly

Reply:
not sure I got you. do you mean `Create a network attachment definition:`?